### PR TITLE
[Rule Tuning] Suspicious JAR Child Process

### PR DESCRIPTION
**Pull request by Samirbous**
Expand rule coverage by removing the process.args containing a jar file requirement which may help detect also exploitation attempt via command injection vulnerabilities on server apps running JAVA (e.g. JDBC-Attack).

### CODE REVIEW

**Review by brokensound77 — APPROVED** [review]
This LGTM, but instead of deleting the old tuning logic, we could just comment it out with a comment stating the additional logic contains tuning suggestions as needed, based on environment. Thoughts?

File: rules/cross-platform/execution_suspicious_jar_child_process.toml [hunk 1]
```diff
@@ -24,11 +24,7 @@ type = "eql"
 query = '''
 process where event.type in ("start", "process_started") and
   process.parent.name : "java" and
-  process.name : ("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget") and
-  process.args : "-jar" and process.args : "*.jar" and
-  /* Add any FP's here */
-  not process.executable : ("/Users/*/.sdkman/*", "/Library/Java/JavaVirtualMachines/*") and
-  not process.args : ("/usr/local/*", "/Users/*/github.com/*", "/Users/*/src/*")
+  process.name : ("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget")
```
Comment (brokensound77) [inline rules/cross-platform/execution_suspicious_jar_child_process.toml:27]:
```suggestion
  process.name : ("sh", "bash", "dash", "ksh", "tcsh", "zsh", "curl", "wget")
  
  /* optional tuning opportunities (previously included in rule)
    /* inclusions */
    process.args : "-jar" and process.args : "*.jar"
    
    /* exclusions */
    not process.executable : ("/Users/*/.sdkman/*", "/Library/Java/JavaVirtualMachines/*") and
    not process.args : ("/usr/local/*", "/Users/*/github.com/*", "/Users/*/src/*")
  */
```

**Comment by jtnk** [comment]
👍  initial check looks OK - may need further tuning for wget / curl next week.

**Comment by tonymeehan** [comment]
Should we also change the name so that people don't think JAR is a core part of the detection now that it's optional?

**Comment by tonymeehan** [comment]
Perhaps we could reference log4j2 so that people weeks from now will know to use this?

**Comment by Samirbous** [comment]
image

File: etc/version.lock.json [hunk 2]
```diff
@@ -1648,7 +1648,7 @@
     "version": 4
   },
   "8acb7614-1d92-4359-bfcf-478b6d9de150": {
-    "rule_name": "Suspicious JAR Child Process",
+    "rule_name": "Suspicious JAVA Child Process",
```
Comment (brokensound77) [inline etc/version.lock.json:1651]: You can undo this change - this file is only updated programmatically during the lock process

Comment (brokensound77) [reply etc/version.lock.json:1651]:
```suggestion
    "rule_name": "Suspicious JAR Child Process",
```

**Pull request merged**